test/check-machines-settings

```diff
@@ -272,6 +272,11 @@ class TestMachinesSettings(VirtualMachinesCase):
         # Open dialog
         b.click("#vm-subVmTest1-boot-order button")
         b.wait_visible("#vm-subVmTest1-order-modal-window")
+        # Check some details of boot options
+        b.wait_in_text("#vm-subVmTest1-order-modal-device-row-0 div.list-view-pf-main-info span",
```

Comment:
Can you add in the comment what kind of 'details' are these? That or add an id to the JS code that makes clear what that selector represents and replace the selector here with the id.

Reply:
OK, I add comment there, and I will sperate